frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1716,44 +1610,33 @@ const ManageHostsPage = ({
     return false;
   };
 
-  const renderSidePanel = () => {
-    let SidePanel;
+  const renderCustomControls = () => {
+    // we filter out the all hosts label as we dont want to display it as a
+    // the selected label in the label filter
+    const selectedDropdownLabel =
+      selectedLabel?.name !== "All Hosts" ? selectedLabel : undefined;
```

Comment:
When playing with the UI I was wondering if maybe we do want "All Hosts" listed. Otherwise, the only way to get back to all hosts is to close the filter pill, which is easy to miss.

@noahtalerman do you have any thoughts on including "All hosts" in the new filter dropdown?

Reply:
Good question. The way this is built is that the All hosts is in the other select dropdown filter. So if we do include it it will be in both in the label filter and status filter.

Reply:
> do you have any thoughts on including "All hosts" in the new filter dropdown?

@lukeheath I prefer to leave "All hosts" out of the dropdown.

We removed "All hosts" because it doesn't behave as expected. "All hosts" navigates me to the **Hosts** page with all filters cleared. I would expect "All hosts" in the dropdown to only clear the platform/label filter.

It looks like we have the same confusing behavior in the status (online/offline) dropdown. I tracked this in UI/UX office huddle doc: https://docs.google.com/document/d/1BOtAdyV5k5ne07qVxbblLZ3AY_O58oE01elZu_aqzgU/edit#